HW2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="index.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<div>
+				<img src="logo.png">				
+			</div>
+				<menu class="menu">
+					<li><a class="menu__link" href="#">Главная</a></li>
+					<li><a class="menu__link" href="#">Товары</a></li>
+					<li><a class="menu__link" href="#">О_нас</a></li>
+					<li><a class="menu__link" href="#">Контакты</a></li>
+				</menu>
+		</header>
+		<div class="container">
+				<menu class="left__menu">
+					<li><a class="left__link" href="">Ноутбуки</a></li>
+					<li><a class="left__link" href="">Планшеты</a></li>
+					<li><a class="left__link" href="">Телефоны</a></li>
+					<li><a class="left__link" href="">Телевизоры</a></li>
+					<li><a class="left__link" href="">Бытовая техника</a></li>
+					<li><a class="left__link" href="">Автотовары</a></li>					
+				</menu>				
+			<article class="article__lorem">
+				<div class="article__photo">
+					<img class="photo" src="image.png">	
+				</div>
+				<div class="article__description">Lorem Ipsum - це текст -"риба", що використовується в друкарстві та дизайні. Lorem Impus є, фактично, стандартною "рибою" аж з XVI сторіччя, коли невідомий друкар взяв шрифтову гранку та склав на ній підбірку зразків шрифтів. "Риба" не тільки успішно пережила п'ять століть, але й прижилася в електронному верстуванні, залишаючись по суті незмінною. Вона популяризувалась в 60-тих роках минулого сторіччя завдяки виданню зразків шрифтів Letraset, які містил уривки з Lorem Impus, і вдруге - нещодавно завдяки програмам комп'ютерного верстування на кшталт Aldus Pagemaker, які використовували різні версії Lorem Ipsum.				
```

Comment:
wtf? have you read my comment about invalid div that you forget to close in previous review?